pyproject.toml

```diff
@@ -10,3 +10,23 @@ requires = [
     "numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'",
     "numpy==1.16.0; python_version>='3.7' and platform_system=='AIX'",
 ]
+
+[tool.black]
+target-version = ['py36', 'py37', 'py38']
```

Comment:
Looking over the `black` issue tracker it looks like we should be specifying all versions instead of just the lower bound.  The logic being that a new version of Python could change the syntax such that a kind of formatting that `black` thinks is ideal is no longer supported.

Reply:
is there a reason for pyproject.toml instead of setup.cfg?

Reply:
`black` does not support configuration through `setup.cfg` nor do they plan on supporting it in the future